x-pack/plugins/cloud_security_posture/public/components/no_findings_states.tsx

```diff
@@ -116,6 +125,43 @@ const IndexTimeout = () => (
   />
 );
 
+const Unprivileged = ({ unprivilegedIndices }: { unprivilegedIndices: string[] }) => (
+  <EuiEmptyPrompt
+    data-test-subj={NO_FINDINGS_STATUS_TEST_SUBJ.UNPRIVILEGED}
+    color="plain"
+    icon={<EuiIcon type="logoSecurity" size="xl" />}
+    title={
+      <h2>
+        <FormattedMessage
+          id="xpack.csp.noFindingsStates.unprivileged.unprivilegedTitle"
+          defaultMessage="Privileges required"
+        />
+      </h2>
+    }
+    body={
+      <p>
+        <FormattedMessage
+          id="xpack.csp.noFindingsStates.unprivileged.unprivilegedDescription"
+          defaultMessage="To view cloud posture data, you must update privileges. For more information, contact your Kibana administrator."
+        />
+      </p>
+    }
+    footer={
+      <EuiMarkdownFormat
+        css={css`
+          text-align: initial;
+        `}
+        children={
+          i18n.translate('xpack.csp.noFindingsStates.unprivileged.unprivilegedFooterMarkdown', {
+            defaultMessage:
+              'Required Elasticsearch index privilege `read` for the following indices:',
+          }) + unprivilegedIndices.map((idx) => `\n- \`${idx}\``)
+        }
```

Comment:
if `unprivilegedIndices` is empty we probably don't want to show any text.

Reply:
I agree
But it seems like common sense to use `Unprivileged` component and pass the relevant indices, wdyt?